x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/enrich/20_standard_index.yml

```diff
@@ -187,3 +187,61 @@ enrich documents over _bulk via an alias:
   - do:
       enrich.delete_policy:
         name: test_alias_policy
+
+---
+enrich stats REST response structure:
```

Comment:
@masseyke @joegallo I added a YAML REST test that validates the response structure of the `_enrich/stats` API [0767cc6](https://github.com/elastic/elasticsearch/pull/110578/commits/0767cc65446ff1042c28d26bbb95959824817111). Let me know what you think (also if you think this is even valuable/worth it or not).

Reply:
I'm not sure that it adds any value that enrich-stats.asciidoc doesn't already provide. But it probably doesn't hurt anything, and might be a useful building block to someone in the future. So now that you've written it, might as well leave it in.

Reply:
Ah right, that's true. I forgot that those docs tests already provide some validation on the REST structure.

Reply:
I like having the yaml test, even if there's technically some redundancy with the docs tests.